core/src/main/java/org/mule/api/MuleEvent.java

```diff
@@ -322,6 +322,10 @@
 
     void clearFlowVariables();
 
+    /**
+     * @deprecated use {@code getSession()} to manipulate session state
+     */
+    @Deprecated
```

Comment:
These changes are not related to the changes we discussed about changing the message/event. This is just cleaning things that are not used anymore or must not be used